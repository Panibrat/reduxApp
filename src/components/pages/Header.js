import React from 'react';

export default class Header extends React.Component {
  render() {
    return(
        <h1>Header Enzyme</h1>
    )
  }
}
