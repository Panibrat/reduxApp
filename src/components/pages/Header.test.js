import React from 'react';
//import ReactShallowRenderer from 'react-test-renderer/shallow';
import {shallow} from 'enzyme';
import Header from './Header';

test('should render Header correctly', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper.find('h1').length).toBe(1);
  expect(wrapper.find('h1').text()).toBe('Header');


  // const renderer = new ReactShallowRenderer();
  // renderer.render(<Header />);
  // expect(renderer.getRenderOutput()).toMatchSnapshot();
});
