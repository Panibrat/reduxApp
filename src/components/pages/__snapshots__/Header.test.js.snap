// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<h1>
  Header
</h1>
`;
